taxonomies/categories.txt

```diff
@@ -28969,7 +28951,7 @@ ciqual_food_name:fr:Maïs doux, appertisé, égoutté
 
 <en:Canned vegetables
 <en:White beans
-en:Canned drained haricot beans, Canned haricot beans
+en:Canned haricot beans, Canned haricot beans
```

Comment:
the synonym is the same as the first entry
also the French translation still contains "égouttés".

-> the "appertisés" should just be renamed to "en conserve", and we can make it a synonym:fr:en conserve,appertisé,appertisée,appertisés,appertisées entry